### PR TITLE
ETR - File Browser Update

### PR DESCRIPTION
**Pull request by wtconsulting**
These functions in the FileBrowser / FileViewer are attempting to iterate over the file and image extensions that the user defines in the settings.ini.cfm as filebrowsereditlist etc. However, the iteration is happening over a string.

This image illustrates the issue. Note the "t" in the debug popup where it should be comparing against a whole extension (txt for example).
image

I also updated the CSS to fix the Edit window being cut in half and right justified on the page. 
image

**Note** I made this change in a non-minified JS file. If you need me to minify it or make any updates, I can do that as well. I did test that this resolves the issues.

### CODE REVIEW

File: admin/assets/js/filebrowser/assets/css/filebrowser.css [hunk 1]
```diff
@@ -144,15 +144,15 @@
 
 #actionwindow-wrapper {
   position: absolute;
-  margin: 60px 30%;
+  margin: 30px 30px;
```
Comment (jeroenweareorangeeu) [inline admin/assets/js/filebrowser/assets/css/filebrowser.css:147]: I would revert this change;  the old css makes sure that the popup is always centered in the middle.

Comment (wtconsulting) [reply admin/assets/js/filebrowser/assets/css/filebrowser.css]: @jeroenweareorangeeu Reverted, cheers.

**Review by guustnieuwenhuis — APPROVED** [review]

**Comment by guustnieuwenhuis** [comment]
Thank you @wtconsulting!
